### PR TITLE
Add KDE Q2 report

### PR DESCRIPTION
**Pull request by adriaandegroot**

### CODE REVIEW

File: 2022q2/kde.adoc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+Links: +
+link:https://freebsd.kde.org/[KDE FreeBSD] URL: link:https://freebsd.kde.org/[https://freebsd.kde.org/] +
+link:https://community.kde.org/FreeBSD[KDE Community FreeBSD] URL: link:https://community.kde.org/FreeBSD[https://community.kde.org/FreeBSD]
+
+Contact: Adriaan de Groot <kde@FreeBSD.org>
+
+The KDE on FreeBSD project packages the software from the KDE Community, along with dependencies and related software, for the FreeBSD ports tree.
+The software includes a full desktop environment called KDE Plasma (for both X11 and Wayland) and hundreds of applications that can be used on any FreeBSD machine.
+
+The KDE team (kde@) is part of desktop@ and x11@ as well, building the software stack to make FreeBSD beautiful and usable as a daily-driver graphics-based desktop machine. The notes below describe **mostly** ports for KDE, but also include items of import to the entire desktop stack.
```
Comment (pauamma) [inline 2022q2/kde.adoc:12]:
```suggestion
The KDE team (kde@) is part of desktop@ and x11@ as well, building the software stack to make FreeBSD beautiful and usable as a daily-driver graphics-based desktop machine. The notes below describe *mostly* ports for KDE, but also include items of import to the entire desktop stack.
```

File: 2022q2/kde.adoc [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+==== KDE Stack
+
+KDE Gear releases happen each quarter, KDE Plasma updates once a month, and KDE Frameworks have a new release each month as well. These (large) updates land shortly after their upstream release and are not listed separately.
+
+* *astro/kstars* latest release 3.5.9.
+* *deskutils/grantleetheme* got an entry in UPDATING because of some unusual changes to the installed structure of the port.
+* *deskutils/kalendar* joined the KDE Gear releases.
+* *devel/okteta* updates to the octal viewer and editor.
```
Comment (pauamma) [inline 2022q2/kde.adoc:21]: Clarifying: do you mean "octal" (base-8) here, or "octet" (fancy name for 8-bit bytes)? For reference, in the 2022Q1 report, you called it "an editor and viewer for binary data" and its pkg-descr says "Numerical encodings: Hexadecimal, Decimal, Octal, Binary".

File: 2022q2/kde.adoc [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  releases, well worth looking at if you need any kind of data-plotting.
+* *net-im/ruqola* was updated. This is a Qt-styled Rocket chat application.
+* *www/falkon* joined the KDE Gear releases.
+
+==== Related Applications
+
+* *archivers/quazip* was updated.
+* *deskutils/semantik* updated.
+* *devel/py-qt5-pyqt* updated in an effort to make the port
+  more all-inclusive.
```
Comment (pauamma) [inline 2022q2/kde.adoc:36]: Clarify; "more all-inclusive" of who or what? (and it should probably be "more inclusive" as "all-inclusive" doesn't have degrees").

File: 2022q2/kde.adoc [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+* *devel/qcoro* had build issues on certain FreeBSD versions, resolved.
+* *devel/qtcreator* updated with each new release.
+* *devel/qt5* had its infrastructure updated in ports so it does not
+  produce strange error messages during de-installation.
+* *graphics/ksnip* and related libraries updated to recent releases.
+* Matrix clients Nheko (*net-im/nheko*) and Neochat (*net-im/neochat*)
+  were updated following releases and library bumps.
+* *x11/rsibreak* updated, which helps prevent injury while writing
```
Comment (pauamma) [inline 2022q2/kde.adoc:44]:
```suggestion
* *x11/rsibreak* updated; helps prevent injury while writing
```

File: 2022q2/kde.adoc [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+* *archivers/quazip* was updated.
+* *deskutils/semantik* updated.
+* *devel/py-qt5-pyqt* updated in an effort to make the port
+  more all-inclusive.
+* *devel/qcoro* had build issues on certain FreeBSD versions, resolved.
+* *devel/qtcreator* updated with each new release.
+* *devel/qt5* had its infrastructure updated in ports so it does not
+  produce strange error messages during de-installation.
+* *graphics/ksnip* and related libraries updated to recent releases.
+* Matrix clients Nheko (*net-im/nheko*) and Neochat (*net-im/neochat*)
```
Comment (pauamma) [inline 2022q2/kde.adoc:42]:
```suggestion
* Matrix clients Nheko (*net-im/nheko*) and Neochat (*net-im/neochat*).
```

File: 2022q2/kde.adoc [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  were updated following releases and library bumps.
+* *x11/rsibreak* updated, which helps prevent injury while writing
+  long quarterly reports.
+
+=== Elsewhere
+
+* *devel/appstream* update supports more application-information.
+* *devel/cmake* prefers generic python3 over versioned-python3, if
+  users have multiple python3 ports **and** lang/python3 installed.
```
Comment (pauamma) [inline 2022q2/kde.adoc:51]:
```suggestion
  users have multiple python3 ports *and* lang/python3 installed.
```

**Review by pauamma — CHANGES_REQUESTED** [review]

File: 2022q2/kde.adoc (hunk 5 above)
Comment (adriaandegroot) [reply 2022q2/kde.adoc:42]: No, that line carries on to the next (but I'll join them into one text line).

File: 2022q2/kde.adoc (hunk 6 above)
Comment (adriaandegroot) [reply 2022q2/kde.adoc:51]: I keep forgetting about adoc vs markdown emphasis, yeah.

**Review by pauamma — APPROVED** [review]

**Pull request merged**